modules/web/.husky/pre-push

```diff
@@ -16,4 +16,9 @@
 . "$(dirname "$0")/_/husky.sh"
 cd modules/web/.husky
 
+# If there are no changes in modules/web, exit the hook early so we don't run on BE-only changes.
+if git diff --quiet --cached main -- "$(git rev-parse --show-toplevel)/modules/web"; then
+  exit 0
+fi
+
 npm run test
```

Comment:
```suggestion
if git diff --quiet --cached -- "$(git rev-parse --show-toplevel)/modules/web"
then
  npm run test
else
  exit 0
fi
```

Basically you need to reverse the condition.

Reply:
If I read the git doc correctly ["it exits with 1 if there were differences and 0 means no differences"](https://git-scm.com/docs/git-diff#Documentation/git-diff.txt---exit-code). For our if statement this would mean that if statement was successful (aka exit code 0) there would be no changes, thus implying you can leave early.

Or am I missing something?

Reply:
I misunderstood it completely, you are absolutely right here.